pkg/buildHeaderLineTypeUtilities/buildHeaderLineTypeUtilities_test.go

```diff
@@ -22,6 +23,7 @@ func TestGetLineType(t *testing.T) {
 			dataLine:      "V12.0.0 ECMWF NA   060000    20241031_000000 20241031_000000 000000   20241031_000000 20241031_000000 TMP       K          P1000    TMP       K         P1000   ANLYS  FULL    NEAREST     1           NA          NA         NA         NA    SAL1L2    1038240   0.84332      0.45998         6.19154        7.38574        7.9421     1.02924",
 			fileType:      "STAT",
 			fileName:      "point_stat_360000L_20070331_120000V_sal1l2.txt",
+			version:       "V12.0.0",
 			wantType:      "STAT_SAL1L2",
 			wantHeader:    []string{"V12.0.0", "ECMWF", "NA", "", "1730332800", "1730332800", "000000", "1730332800", "1730332800", "TMP", "K", "P1000", "TMP", "K", "P1000", "ANLYS", "FULL", "NEAREST", "1", "NA", "NA", "NA", "NA", "SAL1L2"},
 			wantData:      []string{"1038240", "0.84332", "0.45998", "6.19154", "7.38574", "7.9421", "1.02924"},
```

Comment:
Do we have a test to confirm that when the code doesn't get a `version` (or we get a `version` we don't support) that the code returns an appropriate error? That seems like an important test case given we'll be accepting user-generated input and there aren't guarantees on what versions of MET they're running.

Reply:
I can add that.